.github/workflows/ci.yaml

```diff
@@ -14,24 +14,21 @@ jobs:
 
         runs-on: ubuntu-latest
 
+        container: python:3.7-alpine
+
         steps:
             -   name: "Checkout"
                 uses: actions/checkout@v2
 
-            -   name: "Set up Python 3.7"
-                uses: actions/setup-python@v1
-                with:
-                    python-version: '3.7' # Semantic version range syntax or exact version of a Python version
-
             -   name: "Display Python version"
                 run: python -c "import sys; print(sys.version)"
 
-            -   name: "Install Sphinx dependencies"
-                run: sudo apt-get install python-dev build-essential
-
             -   name: "Install Sphinx"
                 run: pip install --user sphinx
 
+            -   name: "Install dependencies"
+                run: apk add --no-cache git make
```

Comment:
Can you explain why we need make and git here?

Reply:
It's required for `pip` afaics (git is used to clone the fabpot sphinx extension and make is used by setuptools I believe)